public-api/src/main/java/org/pmiops/workbench/cdr/dao/QuestionConceptDao.java

```diff
@@ -0,0 +1,14 @@
+package org.pmiops.workbench.cdr.dao;
+import org.pmiops.workbench.cdr.model.QuestionConcept;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+
+import java.util.List;
+
+public interface QuestionConceptDao extends CrudRepository<QuestionConcept, Long> {
+
+    @Query(nativeQuery=true, value="SELECT concept_id, concept_name, domain_id, vocabulary_id, concept_code, " +
```

Comment:
Nit: qualify the tables you're using in the select clause... it's unclear what columns are coming from what table this way.

Reply:
Done.